packages/aws/changelog.yml

```
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "1.35.0"
+  changes:
+    - description: Add a new flag to enable request tracing
```

```suggestion
    - description: Add a new flag to enable request tracing on `httpjson` based input
```

I'm not sure my suggestion is the best possible, but we should give a hint about what is getting a new tracing option, WDYT?